.github/workflows/functional-tests-workflow.yml

```diff
@@ -27,6 +27,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DBT_TEST_ATHENA_S3_STAGING_DIR: s3://dbt-athena-query-results-${{ inputs.aws-region }}
+      DBT_TEST_ATHENA_S3_TMP_TABLE_DIR: s3://dbt-athena-query-results-${{ inputs.aws-region }}
```

Comment:
nit: shall we add also a final prefix?
e.g. `s3://dbt-athena-query-results-${{ inputs.aws-region }}/tmp_tables/` ?

Reply:
OK!
I added the prefixes.
[40da3dc](https://github.com/dbt-athena/dbt-athena/pull/434/commits/40da3dc50d58682135419ac95790143ea582464a)